src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml

```
@@ -67,3 +67,11 @@ include:
           - "positive"
       description: >-
           Which side to defend
+
+- int:
+      name: vision_port
+      value: "10006"
+      min: 0
+      max: 100000
```

```suggestion
      min: 1
      max: 65535
```
nit: the valid range of port numbers is from 1 to 65535

---

fixed